redash/settings/__init__.py

```diff
@@ -511,4 +511,7 @@ def email_server_is_configured():
     os.environ.get("REDASH_ENFORCE_CSRF", "false")
 )
 
+# Databricks
+DATABRICKS_ROW_LIMIT = int(os.environ.get("REDASH_DATABRICKS_ROW_LIMIT", 20000))
```

Comment:
Let's move this definition directly into `redash.query_runner.databricks` as we do with other query runner specific settings.